crates/trycp_server/src/main.rs

```diff
@@ -80,11 +82,22 @@ struct TrycpServer {
     port_range: PortRange,
 }
 
+fn make_conductor_dir() -> PathBuf {
+    let conductor_path = PathBuf::new().join(TRYCP_DIRNAME).join(CONDUCTORS_DIRNAME);
+    std::fs::create_dir_all(conductor_path.clone()).expect("can create conductors dir");
```

Comment:
This part should probably be a _create if it doesn't exist already_ pattern.

Reply:
I think that's exactly what create_dir_all does.  If it's already there it won't panic